sdk/attestation/attestation/src/attestationClient.ts

```diff
@@ -6,40 +6,139 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
-import * as coreHttp from "@azure/core-http";
+import {
+  PipelineOptions,
+  bearerTokenAuthenticationPolicy,
+  createPipelineFromOptions,
+  TokenCredential,
+  OperationOptions,
+} from "@azure/core-http"
+//import { OperationOptions } from "@azure/core-client";
+import { SDK_VERSION } from "./constants";
 import {
   Policy,
   PolicyCertificates,
   Attestation,
-  SigningCertificates,
-  MetadataConfiguration
 } from "./operations";
-import { AttestationClientContext } from "./attestationClientContext";
-import { AttestationClientOptionalParams } from "./models";
+import { GeneratedClient } from "./generated/generatedClient"
+  
+import { AttestationSigner } from "./models";
+
+import { logger } from "./logger";
+import { createSpan } from "./tracing";
+import { GeneratedClientOptionalParams } from "./generated/models";
+
+/**
+* Attestation Client Options.
+*/
+export interface AttestationClientOptions extends PipelineOptions{};
 
-export class AttestationClient extends AttestationClientContext {
+export interface AttestationClientOperationOptions extends OperationOptions{};
+
+export class AttestationClient {
   /**
-   * Initializes a new instance of the AttestationClient class.
-   * @param credentials Subscription credentials which uniquely identify client subscription.
+   * Creates an instance of AttestationClient.
+   *
+   * Example usage:
+   * ```ts
+   * import { AttestationClient } from "@azure/security-attestation";
```

Comment:
is this a typor or a new npm name for the package? currently on npm it is called `@azure/attestation`.

Reply:
I thought @azure/attestation was the Track 1 SDK. Fixing it.